common/source/docs/common-autopilots.rst

```diff
@@ -88,4 +92,101 @@ available if you're still working on those platforms:
    PX4FMU
    Qualcomm Snapdragon Flight Kit
 
+.. _common-autopilots_limited_firmware:
+
+Firmware Limitations on Selected Hardware
+-----------------------------------------
+
+The ArduPilot firmware in some configurations exceeds 1 MB in size. Some autopilots may not have enough
+flash memory to store the full firmware.
+
+For the affected autopilots, a minimal firmware is generated. This firmware omits less-commonly used features
+in order to reduce the firmware size to less than 1 MB.
+
+Miminal firmware is required for the KakuteF7, KakuteF7 Mini, OmnibusF7V2, sparky2 and older versions of
```

Comment:
same comment

Reply:
@tridge why is it that some 1MB boards have this greatly reduced feature set, and others much less reduction?
Also, as a maintenance issue, how do we flag when a new feature is added to the MINIMIZE_FEATURES list or a specific board in order to keep the Wiki up to date?

Reply:
This section should and the following section for other boards should be in the same format style.... with the boards bulleted and feature list indented...perhaps for these first ones, first indent is vehicle, then feature...

Reply:
Fixed the wording